docs/provider/secretserver.md

```diff
@@ -131,3 +131,81 @@ returns: The entire secret in JSON format as displayed below
   ]
 }
 ```
+
+### Referencing Secrets in multiple Items secrets
+
+If there is more then one Item in the secret, it supports to retrieve them (all Item.\*.ItemValue) looking up by Item.\*.FieldName or Item.\*.Slug, instead of the above behaviour to use gjon only on the first item Items.0.ItemValue only.
```

Comment:
```suggestion
If there is more then one Item in the secret, it supports to retrieve them (all Item.\*.ItemValue) looking up by Item.\*.FieldName or Item.\*.Slug, instead of the above behaviour to use json only on the first item Items.0.ItemValue only.
```